tracer/src/Datadog.Trace/Debugger/Snapshots/DebuggerSnapshotSerializer.cs

```diff
@@ -233,7 +229,7 @@ private static void WriteFieldsInternal(object source, JsonWriter jsonWriter, Ca
                     continue;
                 }
 
-                if (index == 0 && writeFieldsJsonElement)
+                if (index == 0)
```

Comment:
Among the tests we disabled, are there tests that only have an async line probe but don't have an async method probe?
If yes, could you please bring them back as part of this PR? If not, can you please add one?

Reply:
Restored the test `AsyncInstanceMethod` in f00e4a263667d87af6d7412138c14d172290ef87.